src/main/scala/io/gitlab/agluszak/tickets/http/JsonRejectionHandler.scala

```diff
@@ -0,0 +1,15 @@
+package io.gitlab.agluszak.tickets.http
+
+import akka.http.scaladsl.model.{ContentTypes, HttpEntity, HttpResponse}
+import akka.http.scaladsl.server.RejectionHandler
+
+object JsonRejectionHandler {
+  implicit val jsonRejectionHandler: RejectionHandler =
+    RejectionHandler.default
+      .mapRejectionResponse {
+        case res@HttpResponse(_, _, ent: HttpEntity.Strict, _) =>
+          val message = ent.data.utf8String.replaceAll("\"", """\"""")
+          res.copy(entity = HttpEntity(ContentTypes.`application/json`, s"""{"rejection": "$message"}"""))
```

Comment:
Hand-making string that contains JSON and manually ensuring that double-quotes are properly escaped is error-prone. Couldn't you use some JSON library for that?